definitions/variable/grids infrustructure/electricity_grid.yaml

```
@@ -0,0 +1,90 @@
+- Network|Electricity|Capacity:
+    description: Capacity of the line
+    unit: MW
+- Network|Electricity|Resistance
+    description: Resistance of an line
+    unit: Ohm
+- Network|Electricity|Resistance
+    description: Resistance of an line
+    unit: Ohm
+- Network|Electricity|Reactance:
+    description: Reactance of an line
+    unit: Ohm
+- Network|Electricity|Capacitance
+    description: Parallel capacitance of an line
+    unit: nF
+- Capacity|Electricity|Hydro|Pumped Storage:
+    description: Installed (available) capacity to generate electricity from hydropower pumped storage
+    unit: 'GW'
```

No quotation marks needed unless the unit (or any other attribute) starts with yaml-specific characters.
```suggestion
    unit: GW
```